api/src/services/registration/registration.class.ts

```diff
@@ -0,0 +1,27 @@
+import { BadRequest } from '@feathersjs/errors'
+import pick from 'lodash/pick'
+
+import { Application } from '../../declarations'
+import { User } from '../../models/users.model'
+
+
+export class RegistrationService  {
+
+  app: Application
+
+  constructor(app: Application) {
+    this.app = app
+  }
+
+  create(data : any) : Promise<User>{
```

Comment:
Ideally, we should try not to use `any` unless absolutely necessary. I think it should be safe to do `(data: Partial<User>)` here.